documentation/staging/content/managing-domains/accessing-the-domain/access-admin-console-em.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Access the Administration Console"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 1
+description: "Access the WebLogic Admin Console to manage domains running in Kubernetes."
+---
+
+You can use the WebLogic Server Administration Console to monitor and manage a WebLogic domain running in Kubernetes.
+
+{{% notice note %}} Do not use the WebLogic Server Administration Console to start or stop servers, or for scaling clusters. See [Starting and stopping servers]({{< relref "/managing-domains/domain-lifecycle/startup#starting-and-stopping-servers" >}}) and [Scaling]({{< relref "/managing-domains/domain-lifecycle/scaling.md" >}}). In addition, if your domain home type is either [Domain in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}) or [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}), then do not use the Administration Console to make changes to the WebLogic domain configuration as these changes are ephemeral and will be lost when servers restart. See [Choose a domain home source type]({{< relref "/managing-domains/choosing-a-model/_index.md" >}}).
+{{% /notice %}}
+
+To setup WebLogic Server Administration Console access to a domain running in Kubernetes, you can:
+   * Deploy a load balancer with [ingress path routing rules for non-SSL port](#configure-ingress-path-routing-rules) and [SSL port](#access-the-weblogic-server-administration-console-through-the-ssl-port).
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * [Use a `kubectl port-forward` connection](#use-a-kubectl-port-forward-connection).
+
+{{% notice warning %}}
+Externally exposing administrative, RMI, or T3 capable WebLogic channels
+using a Kubernetes `NodePort`, load balancer,
+port forwarding, or a similar method can create an insecure configuration.
+For more information, see [External network access security]({{<relref "/security/domain-security/weblogic-channels.md">}}).
+{{% /notice %}}
+
+
+#### Configure ingress path routing rules
```

Comment:
(1) This particular section and the following SSL section look closely related. They should be in the same common section.
(2) All sections are missing an introduction sentence or paragraph. You can move the text you have in the overall introduction for each to the individual sections.